torch/csrc/jit/script/tree_views.h

```diff
@@ -800,12 +800,16 @@ struct Const : public Expr {
     return !isFloatingPoint();
   }
   int64_t asIntegral() const {
-    return c10::stoll(subtree(0)->stringValue());
+    try {
+      return c10::stoll(subtree(0)->stringValue());
+    } catch (const std::out_of_range& e) {
+      throw ErrorReport(range()) << "Integral constant out of range "
+                                    "(must fit in a signed 64 bit integer)";
+    }
   }
   double asFloatingPoint() const {
-    char* dummy;
     return torch::jit::script::strtod_c(
-        subtree(0)->stringValue().c_str(), &dummy);
+        subtree(0)->stringValue().c_str(), /*endptr=*/nullptr);
```

Comment:
This change seems unrelated?

Reply:
The docs say that you can pass a null out and it’s not used so `dummy` was unnecessary

Reply:
I know, but it's not related to the stated purpose o this commit.